201-encrypt-running-linux-vm/azuredeploy.json

```diff
@@ -18,7 +18,7 @@
     "diskFormatQuery": {
       "defaultValue": "",
       "metadata": {
-        "description": "the query string used to identify the disks to format and encrypt. This parameter only works when you choose the EncryptionOperation as EnableEncryptionFormat.[{\"scsi\":\"6:0:0:1\",\"name\":\"azuredisk\"},{\"dev_path\":\"/dev/sdx\",\"name\":\"azuredisk2\",\"file_system\":\"ext4\"}], the scsi means the disk scsi number, the name means the mount point name, the file_system is the file system to use."
+        "description": "the query string used to identify the disks to format and encrypt. This parameter only works when you set the EncryptionOperation as EnableEncryptionFormat.[{\"dev_path\":\"/dev/md0\",\"name\":\"encryptedraid\",\"file_system\":\"ext4\"}], which will format /dev/md0, encrypt it and mount it at /mnt/dataraid. This parameter should only be used for RAID devices. The specified device must not have any existing filesystem on it."
```

Comment:
Is the [...] substring in this "description" an example that users can use? If so, please call say, "For example, [...]."